### ISSUE TITLE
What's the GPU size needed for training D0 (and D7)?

### ISSUE THREAD

**Issue by hareld10** [signatrix/efficientdet#54]
Hi,

What's the GPU size needed for training D0 (and D7)?

Best,

**Comment by zylo117** [comment]
For my efficientdet implement, inferring D0 is 1049MB, D7 is 3819MB. 

For training, the size should be 3 times larger. 

One for the model, one for the gradients, one for the optimizer cache.

You can try it here, with D0 to D7 pretrained weights.

https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch
